lib/ansible/executor/task_executor.py

```diff
@@ -337,6 +338,9 @@ def _run_loop(self, items):
             (self._task, tmp_task) = (tmp_task, self._task)
             (self._play_context, tmp_play_context) = (tmp_play_context, self._play_context)
 
+            # update 'general no_log' based on specific nolog
+            no_log = no_log or tmp_task.no_log
```

Comment:
Would this make it so that if any item has no_log, ask items have no_log?  I've got some loops that set no_log only for some items. (Such as lineinfile or ini_file to set fb connection info.)

Reply:
no, it would set no_log for the 'main task', each item still keeps it's own no_log values